packages/features/schedules/components/ScheduleListItem.tsx

```diff
@@ -67,22 +83,55 @@ export function ScheduleListItem({
           <DropdownMenuTrigger asChild className="mr-5">
             <Button type="button" size="icon" color="secondary" StartIcon={Icon.FiMoreHorizontal} />
           </DropdownMenuTrigger>
-          <DropdownMenuContent>
-            <DropdownMenuItem>
-              <Button
-                onClick={() => {
-                  deleteFunction({
-                    scheduleId: schedule.id,
-                  });
-                }}
-                type="button"
-                color="destructive"
-                className="w-full font-normal"
-                StartIcon={Icon.FiTrash}>
-                {t("delete_schedule")}
-              </Button>
-            </DropdownMenuItem>
-          </DropdownMenuContent>
+          {!isLoading && data && (
+            <DropdownMenuContent>
+              <DropdownMenuItem>
+                <Button
+                  type="button"
+                  color="secondary"
+                  className={classNames("border-none", schedule.isDefault && " bg-white opacity-30")}
+                  disabled={schedule.isDefault}
+                  onClick={(e) => {
+                    if (schedule.isDefault) {
+                      e.preventDefault();
+                    } else {
+                      updateDefault({
+                        scheduleId: schedule.id,
+                        isDefault: true,
+                        schedule: data.availability,
+                      });
+                    }
+                  }}>
+                  <Label
+                    htmlFor="hiddenSwitch"
+                    className={classNames(
+                      "mt-2 mr-2 self-center sm:inline",
+                      schedule.isDefault ? "cursor-not-allowed" : "cursor-pointer"
+                    )}>
+                    {t("set_to_default")}
+                  </Label>
+                  <Switch checked={schedule.isDefault} id="hiddenSwitch" disabled={schedule.isDefault} />
```

Comment:
We should not have switches inside dropdown

Reply:
Should it be just text like in the screenshot? cc: @Jaibles 
<img width="260" alt="Screenshot 2022-11-29 at 1 11 59 PM" src="https://user-images.githubusercontent.com/1501599/204465337-b6317740-7318-4ad4-95d2-9b2c1ae908a5.png">
<img width="233" alt="Screenshot 2022-11-29 at 1 16 30 PM" src="https://user-images.githubusercontent.com/1501599/204466153-15a7ed93-1ce3-4159-ad15-94e5d32ff0d5.png">

Reply:
@Jaibles The implementation is as per spec, but UX wise it just doesn't quite work:
https://www.loom.com/share/4d65f3fad42c4fd4b07d16848f05d0f1

Reply:
Does this need a redesign or is it enough to just remove the switch button?

Reply:
@gitstart let's remove the toggle, the default schedule should not have a "Set as default" button at all. Let's just show the button on the non default ones.